src/components/Picker/index.js

```diff
@@ -102,10 +104,28 @@ class Picker extends PureComponent {
         };
 
         this.onInputChange = this.onInputChange.bind(this);
+        this.placeholder = this.props.placeholder;
+        this.items = this.props.items;
     }
 
     componentDidMount() {
         this.setDefaultValue();
+
+        // Windows will reuse the text color of the select for each one of the options
+        // so we might need to color accordingly so it doesn't blend with the background.
+        if (getOperatingSystem() === CONST.OS.WINDOWS) {
```

Comment:
NAB I think that other platforms were unaffected by the dark theme change and override these styles, so checking for Windows might be unnecessary. That being said, it doesn't hurt to do so.

Reply:
Yeah, that is what I thought and I was going to leave it without it but I didn't want to cause a regression.

Reply:
Makes sense. Let's keep it this way

Reply:
Having the OS check here is the same as having a platform-specific check and our README shares pretty strong opinions about adding them:

> Accordingly, don't even bother writing a platform-specific code block because you're just going to need to undo it.

Can you please take the time to remove this? My guess is the same as Carlos's and that having the color there isn't going to affect the other platforms, but let's be sure to test for regressions.